GIBSDownloader/tile_utils.py

```diff
@@ -19,12 +32,17 @@
 from GIBSDownloader.handling import Handling
 from GIBSDownloader.file_metadata import TiffMetadata, IntermediateMetadata
 from GIBSDownloader.coordinate_utils import Coordinate, Rectangle
+from GIBSDownloader.log_utils import TqdmToLogger
+from GIBSDownloader import log
 
 warnings.simplefilter('ignore', Image.DecompressionBombWarning) # ignore image size warnings
 
 MAX_INTERMEDIATE_LENGTH = int(math.sqrt(2 * Image.MAX_IMAGE_PIXELS)) # Maximum width and height for an intermediate tile to guarantee num pixels less than PIL's max
 NUM_CORES = multiprocessing.cpu_count()
 
+# Redirect tqdm output to logger
+TQDM_OUT = TqdmToLogger(log,level=logging.INFO)
```

Comment:
I'm wondering if this can be initialised via configuration (such as from `__init__.py`) so that if the user wants to make any changes to the logging level, they could all be done from one place.